### PR TITLE
Added slides for lesson 14

### PR DESCRIPTION
**Pull request by alena456**

### CODE REVIEW

File: src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+      <section>
+        <p>Hey! 👋🏼</p>
+        <h1 class="r-fit-text">Form Validation + Recap Dom manipulations</h1>
+        <p>Redi School Munich - Spring 2021</p>
+      </section>
+
+      <section class="initial">
+        <section>
+          <h2>Recap on DOM Manipulation</h2>
+          <a href="https://github.com/alena456">Kahoot!</a>
```
Comment (Stephs15) [inline src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html]: Is this supposed to be your github profile? 👀

**Comment by Stephs15** [comment]
Are we giving them also homework related to their projects? Maybe we can suggest they implement a contact me form for their project 🤔

File: src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+          <code class="formattedCode html" data-trim>
+            <style>
+              .pseudo_example:invalid {
+                border: 2px dashed red;
+              }
+
+              .pseudo_example:valid {
+                border: 2px solid green;
+              }
+
```
Comment (Stephs15) [inline src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html:318]: I would add that showing messages about the error is also a possibility and should be encouraged

File: src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html (hunk 1 above)
Comment (alena456) [reply src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html]: I still have to update the Link, thanks for the hint!

File: src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html (hunk 2 above)
Comment (alena456) [reply src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html:318]: Good hint, I will add that

**Comment by alena456** [comment]
I will give them the homework as inspiration with the hint that they should implement this or something  similar for their project, as  discussed yesterday

File: src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html [hunk 3]
```diff
@@ -0,0 +1,419 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Form Validation + Recap Dom manipulations</title>
+  <meta name="date" content="2021-04-26" />
```
Comment (guicheffer) [inline src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html:7]:
```suggestion
  <meta name="date" content="2021-05-20" />
```

**Review by guicheffer — APPROVED** [review]
Looks good from my point of view 🙂

**Review by Stephs15 — APPROVED** [review]

**Review by oktupol — APPROVED** [review]
Looks good to me, apart from the date 👍🏽

**Pull request merged**